rails/config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+Camyyo3fC6OEpeNenhgKROsbCgtEmALMvKiTSqKe1yf+gsm9/EvY/XeHNuAkXmsfXLk28ugH/meYsDtWnXCheUPlaSLgYT05i+BiQglqLLLH4jMUttwWLvj+Kw3813mazLYf1nvm7ZzC/8CXgxxJIWZqiuzHWYCWGu0AIHYTebMi/IVEhfxJ5AieFew9NcVzAf7ncrwpdRff5KmblglrGW8n3ioRnl5LQSc4gozUUBlYsUcsY/vFY0MovedW8naowJqOozRbYRs7Ht9J33hlDjyRoRE7w51NZ8zJxGIDJpvSEyJfOq6K/MPDOEgl3+Sn30L7Aw4LZ7Hn4Qpet6oDG8HbGcXz7fSQKRGr44+XMzzM/dz5JdWXaI1F9+rUKXC/XUc+I416S2UhJc0G/Mdy4E7ysN2vj9zpaUBl--mQiUkwdYdQkeN2G2--Uc2Cr9mrs/UHTsa880ynPw==
```

Comment:
Is this supposed to be included? I don't know what it's for and it looks like maybe something that shouldn't be committed to the codebase, though I'm not sure. Please add a note about why it's included so I can know what I'm signing off on. Thanks.